tools/Google.Cloud.Tools.ReleaseManager/CreatePipelineStateCommand.cs

```diff
@@ -109,4 +205,37 @@ private class ApiGenerationState
         [JsonProperty("automationLevel")]
         public string AutomationLevel { get; set; }
     }
+
+    private class LibraryReleaseState
+    {
+        [JsonProperty("id")]
+        public string Id { get; set; }
+
+        [JsonProperty("currentVersion")]
+        public string CurrentVersion { get; set; }
+
+        [JsonProperty("nextVersion")]
+        public string NextVersion { get; set; }
```

Comment:
Oh, interesting. So librarian reads this from here? Still not sure who suggests a value for it...